clients/go/admin/client.go

```diff
@@ -192,6 +192,11 @@ func initializeClients(ctx context.Context, cfg *Config, tokenCache pkce.TokenCa
 		opts = append(opts, authOpt)
 	}
 
+	if cfg.Balancer != "" {
```

Comment:
nit: maybe `BalancerName`?

Reply:
changed it to LoadBalancingPolicy, what do you think?